types/kagekiri.d.ts

```diff
@@ -8,3 +8,5 @@
 export function querySelector(selector: string, context?: Node): Element | null;
 export function querySelectorAll(selector: string, context?: Node): Element[];
 export function getElementsByClassName(names: string, context?: Node): Element[];
+export function getElementsByTagName(tagName: string, context?: Node): Elements[];
```

Comment:
Looks like a typo? Should be `Element[]`. Same as below.

Reply:
Yeah good catch